src/main/java/org/kainos/ea/db/DatabaseConnector.java

```diff
@@ -0,0 +1,65 @@
+package org.kainos.ea.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public final class DatabaseConnector {
+    public static final String ENV_DB_USERNAME = "DB_USERNAME";
+    public static final String ENV_DB_PASSWORD = "DB_PASSWORD";
+    public static final String ENV_DB_HOST = "DB_HOST";
+    public static final String ENV_DB_NAME = "DB_NAME";
+
+    private static Connection connection;
+
+    public static void resetConnection() {
+        DatabaseConnector.connection = null;
+    }
+
+    private DatabaseConnector() {
```

Comment:
What is this for?

Reply:
> What is this for?

I'm using DatabaseConnector like a static utility class so it's not supposed to be instantiated (the linter I'm using suggested adding an empty private constructor)